src/scripts.js

```diff
@@ -156,6 +156,11 @@ function generateIngredients(recipe) {
   }).join(", ");
 }
 
+function generateRecipeCost(recipe) {
```

Comment:
This is awesome!  Would have never though that some wing sauce would cost me over $150 to make!